src/App.tsx

```diff
@@ -3,33 +3,51 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+  movies: Movie[],
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+    movies: [...moviesFromServer],
+  };
 
   render() {
+    const { query } = this.state;
+    const movies = [...this.state.movies].filter(
```

Comment:
no need to make copy for filter, it doesn't mutate array anyway

```suggestion
    const filteredmMovies = moviesFromServer.filter(
```